orttraining/orttraining/python/training/api/lr_scheduler.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# lr_scheduler.py
+
+from onnxruntime.capi import _pybind_state as C
+
+
+class LRScheduler:
+    """
+    Class for running Learning Scheduler Step in Training.
+    This class is a wrapper of LRScheduler Class.
+    """
+
+    def __init__(self, optimizer, warmup_step_count, total_step_count) -> None:
```

Comment:
Do we not need to set the initial learning rate? Let's keep the behavior consistent with the C training apis.